hack/generator/pkg/codegen/storage/storage_type_factory.go

```diff
@@ -61,38 +82,32 @@ func (f *StorageTypeFactory) Types() (astmodel.Types, error) {
 	return f.outputTypes, nil
 }
 
+// process carries out our transformations
+// Each step reads from outputTypes and puts the results back in there
 func (f *StorageTypeFactory) process() error {
 
-	f.outputTypes = make(astmodel.Types)
+	f.outputTypes = f.referenceTypes.Copy()
```

Comment:
minor: Here with the `outputTypes` nomenclature I kinda liked the `inputTypes` which matched up?

Reply:
Simplified.